index.html

```diff
@@ -8,6 +8,52 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <header class="header">
+      <div class="logo-section">
+        <a href="/" class="logo">Rose</a>
+      </div>
+      <div class="bars">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+
+      <nav class="nav">
+        <ul>
+          <li><img src="./images/Icon-Cancel.svg" class="close-icon" alt="Close icon"></li>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Good job using the `<nav>` tag for menu of the desktop version 👍👍 However, as suggested by the reviewer [here](https://github.com/rosemutai/My-Portfolio/pull/1#discussion_r1108155261), **_please consider wrapping all of the navigation elements inside  a `<nav>` tag, especially the menu_**. Please make sure that the hamburger icon also has a `<nav>` tag as this is the menu for the mobile devices. 
As an example, you can use something like below 🔽
```
<header>
  <div>
    <a href="#" >My Logo</a>
    <nav>
      <ul>
        <li>hamburger icon</li>
      </ul>
      <ul>
        <li>close button</li>
        <li><a href="#portfolio">Portfolio</a></li>
        <li><a href="#about">About</a></li>
        <li><a href="#contact" >Contact</a></li>
      </ul>
    </nav>
  </div>
</header>
```
**_PS: Please note that this is just an example, you can use your own code 😄_**